src/internals/StreamHttpReply.hpp

```diff
@@ -11,9 +11,10 @@
 #define __ArduinoHttpServer__StreamHttpReply__
 
 #include <Arduino.h>
-
 #include "ArduinoHttpServerDebug.h"
 
+#define MAX_HEADERS  3
```

Comment:
Can you change it to modern C++ typesafe constexpr?

Reply:
I can give it a shot, but I haven't used those ever (I am not a native C++ speaker).